easybuild/easyconfigs/f/FFmpeg/FFmpeg-3.4.2-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'ConfigureMake'
+
+name = 'FFmpeg'
+version = '3.4.2'
+
+homepage = 'https://www.ffmpeg.org/'
+description = """A complete, cross-platform solution to record, convert and stream audio and video."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
```

Comment:
@RvDijk Since `FFmpeg` is (or can be) and end user application that is performance-critical, I would stick to building this with `foss` or `intel`?

Reply:
@boegel good point, I was doubting whether GCCcore would be the right choice here, since it can be used for both intel and foss dependencies. I'll change to foss.